### PR TITLE
Add Bounty Context to Treasury Page

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

File: docs/learn/learn-guides-bounties.md [hunk 1]
```diff
@@ -32,6 +34,16 @@ proposal as an OpenGov referendum, the bounty will show as "funded" at the end o
 [Bounties page](https://polkadot.js.org/apps/#/bounties). You can then proceed with assigning
 curators to the bounty.
 
+To minimize storage on chain in the same way as any proposal, bounties don't contain contextual
+information. When a user submits a bounty spending proposal, they will need to find an off-chain way
```
Comment (anaelleparity) [inline docs/learn/learn-guides-bounties.md]:
```suggestion
information. When a user submits a bounty spending proposal, they will need to find an off-chain medium
```

File: docs/learn/learn-guides-bounties.md [hunk 2]
```diff
@@ -32,6 +34,16 @@ proposal as an OpenGov referendum, the bounty will show as "funded" at the end o
...
+to explain the proposal through for example [Polkassembly](https://polkadot.polkassembly.io/opengov)
```
Comment (anaelleparity) [inline docs/learn/learn-guides-bounties.md]:
```suggestion
to explain the proposal, for example a bounty proposal document on Polkassembly [Polkassembly](https://polkadot.polkassembly.io/opengov)
```

File: docs/learn/learn-guides-bounties.md [hunk 3]
```diff
@@ -32,6 +34,16 @@ proposal as an OpenGov referendum, the bounty will show as "funded" at the end o
...
+or [Subsquare](https://polkadot.subsquare.io/).
+[This template](https://docs.google.com/document/d/1-IBz_owspV5OcvezWXpksWDQReWowschD0TFuaVKKcU/edit?usp=sharing)
+can help as a checklist of all needed information for OpenGov voters to make an informed decision.
```
Comment (anaelleparity) [inline docs/learn/learn-guides-bounties.md]:
```suggestion
can be used to submit all the information needed by OpenGov voters to make an informed decision.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 4]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+[assign a curator](./learn-guides-bounties.md#assign-a-curator-to-a-bounty) to a bounty, as well as
+[creating and awarding child bounties](./learn-guides-bounties.md#create-and-award-child-bounties)
+and [claiming child bounties rewards](./learn-guides-bounties.md#claim-a-child-bounty-reward).
+
+:::
+
+### Parent Bounties
+
+Getting treasury funding through OpenGov, depending on which treasury track you submit your
+referendum, can be a long and uncertain process. This can be unideal, for example, for events'
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
referendum, can be a long and uncertain process. This is not always a suitable option, for example, for events'
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 5]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+organizers needing to pay costs upfront or close to the event's date. Bounties solve this problem by
+allowing access to treasury funds once to fund multiple events later on through
+[child bounties](#child-bounties) (that is why bounties are also called _parent_ bounties).
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
[child bounties](#child-bounties). This is why bounties are also called _parent_ bounties.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 6]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+
+Parent bounty proposals aim to access treasury funds once to fund multiple smaller proposals later
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
Parent bounty proposals aim to reserve a portion of treasury funds once that will be used later on.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 7]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+on without creating OpenGov referenda for each one of them. Bounties are managed by curators, where
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
They save proponents the time needed to create and obtain approval for several OpenGov referenda. Bounties are managed by curators, where
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 8]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+usually a curator is a [multi-signature account](./learn-account-multisig.md). Bounties can access a
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
the curator is usually a [multi-signature account](./learn-account-multisig.md). Bounties can access a
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 9]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+promote events, fix a bug or vulnerability, develop a strategy, or monitor a set of tasks related to
+a specific topic: all for the benefit of the
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} ecosystem.
+
+A proposer can [submit a bounty proposal](./learn-guides-bounties.md#submit-a-bounty-proposal) to
+OpenGov,
+[with a curator to be defined later](./learn-guides-bounties.md#assign-a-curator-to-a-bounty), whose
+background and expertise is such that they can determine when the task is complete.
+
+When submitting the value of the bounty, the proposer includes a fee paid to curators willing to
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
When submitting the value of the bounty, the proposer can specify a fee that will be paid to curators willing to
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 10]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+invest their time and expertise in the task: this amount is included in the total value of the
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
invest their time and expertise in the task: this amount will be included in the total value of the
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 11]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+bounty. In this sense, the curator's fee can be defined as subtracting the value paid to the bounty
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
bounty. In this sense, the curator's fee can be defined as the difference between the amounts paid to child bounty
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 12]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+rewardee from the total value of the bounty.
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
awardees and the total value of the bounty.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 13]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+
+Curators are selected through OpenGov referendum after the bounty proposal passes and need to add an
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
Curators are selected through OpenGov referendum after the bounty proposal passes; and they need to pay an
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 14]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+upfront deposit to take the position. The deposit is calculated by multiplying the curator fee by
+{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMultiplier" defaultValue={500000} filter="permillToPercent"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMultiplier" defaultValue={500000} filter="permillToPercent"/> :kusama }}%,
+and it can range between a minimum of
+{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMin" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMin" defaultValue={3333333330} filter="humanReadable"/> :kusama }}
+and a maximum of
+{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMax" defaultValue={2000000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMax" defaultValue={166666666500} filter="humanReadable"/> :kusama }}.
+This deposit can be used to punish them if they act maliciously. However, if they are successful in
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
This deposit can be used to punish curators if they act maliciously. However, if they are successful in
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 15]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+their task of getting someone to complete the bounty work, they will receive their deposit back and
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
managing the bounty to completion, they will receive their deposit back and
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 16]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+part of the bounty reward as a fee payment for their job.
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
part of the bounty funding as a payment for their job.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 17]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+
+Curators are expected to have a well-balanced track record related to the issues the bounty tries to
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
Curators are expected to have a well-balanced track record in addressing the issues that the bounty wants to
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 18]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+resolve: they should be at least knowledgeable on the topics the bounty touches and show project
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
solve. At the very least, they should be very knowledgeable on the topics covered by the bounty and have proven project
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 19]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+management skills or experience. These recommendations ensure an effective use of the mechanism. A
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
management skills or experience. These recommendations help ensure an effective use of the bounty mechanism.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 20]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+Bounty is a reward for a specified body of work - or specified set of objectives - that needs to be
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
A Bounty is a reward for a specified body of work - or specified set of objectives - that needs to be
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 21]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+executed for a predefined treasury amount to be paid out. The responsibility of assigning a payout
+address once the specified set of objectives is completed is delegated to the curator.
+
+The bounty has a predetermined duration of
+{{ polkadot: <RPC network="polkadot" path="consts.bounties.bountyUpdatePeriod" defaultValue={1296000} filter="blocksToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.bountyUpdatePeriod" defaultValue={1296000} filter="blocksToDays"/> :kusama }}
+days with the possibility of being extended by the curator. Aiming to maintain flexibility on the
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
days, with possible extension(s) to be requested by the curator. To maintain flexibility during the
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 22]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+tasks’ curation, the curator will be able to create child bounties for more granularity and
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
tasks’ curation, the curator will also be able to create child bounties for more granularity in the allocation of funds and 
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 23]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+allocation in the next iteration of the mechanism.
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
as part of a nested iteration of the bounty mechanism.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 24]
```diff
@@ -116,3 +116,71 @@ requires a decision deposit of {{ polkadot: 10 DOT :polkadot }}{{ kusama:  0.33
...
+
+### Child Bounties
+
+Child bounties are spawned from [parent bounties](#parent-bounties). Child bounties access funds
```
Comment (anaelleparity) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
Child bounties are spawned from [parent bounties](#parent-bounties). Child bounties are used to access funds
```

**Review by anaelleparity — CHANGES_REQUESTED** [review]
Part 1/2 of the review process.

**Comment by filippoweb3** [comment]
@anaelleparity I accepted all changes from your side. Let me know if this PR is good to go :)

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 25]
```diff
@@ -78,3 +78,71 @@ track on-chain. Learn how to submit a treasury proposal for referendum
...
+{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMultiplier" defaultValue={500000} filter="permillToPercent"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMultiplier" defaultValue={500000} filter="permillToPercent"/> :kusama }}%,
+and it can range between a minimum of
+{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMin" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMin" defaultValue={3333333330} filter="humanReadable"/> :kusama }}
+and a maximum of
+{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMax" defaultValue={2000000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMax" defaultValue={166666666500} filter="humanReadable"/> :kusama }}.
+This deposit can be used to punish curators if they act maliciously. However, if they are successful in
+managing the bounty to completion, they will receive their deposit back and
+part of the bounty funding as a payment for their job.
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
part of the bounty funding as a payment for their efforts.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 26]
```diff
@@ -78,3 +78,71 @@ track on-chain. Learn how to submit a treasury proposal for referendum
...
+
+Curators are expected to have a well-balanced track record in addressing the issues that the bounty wants to
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
Curators are expected to have a decent track record in addressing the issues that the bounty wants to
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 27]
```diff
@@ -78,3 +78,71 @@ track on-chain. Learn how to submit a treasury proposal for referendum
...
+solve. At the very least, they should be very knowledgeable on the topics covered by the bounty and have proven project
+management skills or experience. These recommendations help ensure an effective use of the bounty mechanism.
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
solve. They should be very knowledgeable on the topics covered by the bounty and have proven project
management skills or experience. These recommendations help ensure an effective use of the bounty mechanism.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 28]
```diff
@@ -78,3 +78,71 @@ track on-chain. Learn how to submit a treasury proposal for referendum
...
+A Bounty is a reward for a specified body of work - or specified set of objectives - that needs to be
+executed for a predefined treasury amount to be paid out. The responsibility of assigning a payout
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
executed for a predefined treasury amount that is designated to be paid out. The responsibility of assigning a payout
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 29]
```diff
@@ -78,3 +78,71 @@ track on-chain. Learn how to submit a treasury proposal for referendum
...
+[creating and awarding child bounties](./learn-guides-bounties.md#create-and-award-child-bounties)
+and [claiming child bounties rewards](./learn-guides-bounties.md#claim-a-child-bounty-reward).
+
+:::
+
+### Parent Bounties
+
+Getting treasury funding through OpenGov, depending on which treasury track you submit your
+referendum, can be a long and uncertain process. This is not always a suitable option, for example, for events'
+organizers needing to pay costs upfront or close to the event's date. Bounties solve this problem by
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
referendum, can be a long and uncertain process. This is not always a suitable option, for example, for event
organizers who need to pay costs upfront or close to the event's date. Bounties solve this problem by
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 30]
```diff
@@ -78,3 +78,71 @@ track on-chain. Learn how to submit a treasury proposal for referendum
...
+allowing access to treasury funds once to fund multiple events later on through
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
procuring access to treasury funds in a single shot and use them to fund multiple events later on through
```

**Review by DrW3RK — APPROVED** [review]
some minor fixes enhancing readability.

**Pull request merged**